rts/Sim/Units/CommandAI/MobileCAI.cpp

```diff
@@ -615,9 +615,10 @@ void CMobileCAI::ExecuteGuard(Command& c)
 		return;
 	}
 
+	constexpr int retaliationTimeout = 40;
 	const bool pushAttackCommand =
 		owner->unitDef->canAttack &&
-		(guardee->lastAttackFrame + 40 < gs->frameNum) &&
+		(guardee->lastAttackFrame + retaliationTimeout > gs->frameNum) &&
```

Comment:
Someone reported it was broken on Discord, showed the demo and I was able to quick-fix it.

P.S. Can we bury ancient Mantis in favor of github issues?

Reply:
I have no objection to that, we just have to automate exporting the tickets (at least all the ones still open) to github.

Reply:
I'm not sure it's worth the effort.

Mantis is so polluted with mix of old news and issues no one is going to fix.
I feel it should be just made read-only in case someone will need to dig old some issue.
If something is pressing was left off, people will report it on github.